src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IncentiveReview.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Incentive review information for a prisoner")
+data class IncentiveReview(
+  @Schema(description = "Prisoner number", example = "A1234BC")
+  val prisonerNumber: String,
+  @Schema(description = "Internal reference for a period in prison", example = "1234567")
+  val bookingId: Long,
+
+  @Schema(description = "Prisoner’s first name", example = "John")
+  val firstName: String,
+  @Schema(description = "Prisoner’s last name", example = "Smith")
+  val lastName: String,
+
+  @Schema(description = "Whether the prisoner has an ACCT open alert", example = "true")
+  val acctOpenStatus: Boolean,
+)
+
+@Schema(description = "Incentive reviews list for prisoners at a given location")
+data class IncentiveReviewResponse(
+  @Schema(description = "Prisoner incentive reviews")
+  val reviews: List<IncentiveReview>,
```

Comment:
Interesting so this endpoint responds with a "flat" list, e.g. it doesn't group by IEP level etc 👌

Reply:
it won't ever _group_ by level, it will only return ppl on one level once the filtering is available upstream in offender search (there are separate tickets). i decided to split out the level business into future PRs